src/shared/utils/conversationDisplay.ts

```
@@ -124,3 +128,71 @@ export const shouldShowConversationInPracticeInbox = (
+  lastActivityAt: string | number | Date | null | undefined,
+  isLive: boolean = false
+): ConversationPresence => {
+  // Guard against null/undefined, but allow 0 (Unix epoch)
+  const lastTs = (() => {
+    if (lastActivityAt == null) return 0;
+    const date = lastActivityAt instanceof Date ? lastActivityAt : new Date(lastActivityAt);
+    const ms = date.getTime();
+    return Number.isFinite(ms) ? ms : 0;
+  })();
+  // Compute age; if timestamp is in the future, treat as invalid (not active)
+  let ageMsRaw = Date.now() - lastTs;
```

ESLint

[error] 'ageMsRaw' is never reassigned. Use 'const' instead.

(prefer-const)